cli/exchange/user.go

```diff
@@ -66,18 +69,24 @@ func UserCreate(org, userPwCreds, user, pw, email string, isAdmin bool, isHubAdm
 	// get message printer
 	msgPrinter := i18n.GetMessagePrinter()
 
-	if email == "" {
-		if strings.Contains(user, "@") {
-			email = user
-		} else {
-			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("the email must be specified via -e if the username is not an email address."))
-		}
+	var emailIsOptional bool
+	emailIsOptional, _ = checkExchangeVersionForOptionalUserEmail(org, userPwCreds)
+	fmt.Printf("emailIsOptional: %t\n", emailIsOptional)
+
+	if email == "" && strings.Contains(user, "@") {
+		email = user
+	} else if email == "" && !emailIsOptional {
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("the email must be specified via -e if the username is not an email address."))
 	}
 
 	if isHubAdmin && org != "root" {
 		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, msgPrinter.Sprintf("Only exchange users in the root org can be hubadmins."))
 	}
 
+	if emailIsOptional && !isHubAdmin && org == "root" {
```

Comment:
So `emailIsOptional` means user's exchange version is /or later than 2.61.0. I added my new checks only when their exchange has the version with hubadmin fix, to prevent incompatibility.

Reply:
@dabooz @bmpotter oh I see your point. So this check is also okay for exchange version before 2.61.0. Will remove the version check here